docs/metadata.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Metadata
```

Comment:
Maybe we can rename this to `globalConfig Meta Data` as it is for globalConfig?

Reply:
yea, it might be better title, will change